tests/test_substitution_extensions_extras.py

```diff
@@ -0,0 +1,273 @@
+import subprocess
+import sys
+from pathlib import Path
+from textwrap import dedent
+
+import pytest
+
+from sphinx_substitution_extensions import _exists_dependency
+
+_EXISTS_PROMPT_EXTENSION = _exists_dependency('sphinx-prompt')
+_REASON = 'requires sphinx-prompt to be installed'
+
```

Comment:
Optional - use:

```python
pytestmark = [pytest.mark.skipif(not _EXISTS_PROMPT_EXTENSION, reason=_REASON)]
```

That way all the tests in the file get the `skipif` functionality, including new tests.